app/assets/javascripts/discourse/app/controllers/signup-page.js

```diff
@@ -0,0 +1,469 @@
+import { A } from "@ember/array";
```

Comment:
is this duplicating an existing controller?

Reply:
The `signup-page` naming was used to follow the `login-page` controller, but since there isn't actually a `login` controller, we can rename both